README.md

```diff
@@ -1,14 +1,54 @@
+
 # DexScript - ALPHA
+<img src="https://i.imgur.com/uKfx0qO.png" width="200"> 
 
-DexScript is a set of commands developed by DotZZ on September 14, 2024, used for **BallsDex and CarFigures** Discord bot forks.
-Below is a guide on how to use DexScript, and what DexScript is capable of.
-Currently, DexScript is in alpha. DexScript will have more commands, the ability to support more models, and overall quality-of-life improvements as time passes.
+## What is DexScript?
+DexScript is a set of commands made by ``DotZZ``. The commands simplify things like editing, adding, and deleting things you want in your bot. Basically making your life easier.
 
-<img src="https://i.imgur.com/uKfx0qO.png" width="200"> 
+## What bots can it be used for?
+
+DexScript can be used for BallsDex and CarFigures Instances, each of them has its respective guide on how to install. 
+
+## What commands are there?  
+
+
+So, you are probably wondering what commands are there and for that, there is the list of commands down below. 
+The list of commands is:
+```
+- COUNTRY 
+- SHORT_NAME 
+- CATCH_NAMES 
+- CREATED_AT 
+- REGIME 
+- ECONOMY 
+- HEALTH 
+- ATTACK 
+- RARITY 
+- ENABLED 
+- TRADEABLE 
+- EMOJI_ID 
+- WILD_CARD 
+- COLLECTION_CARD 
+- IMAGE_CREDITS 
+- CAPACITY_NAME 
+- CAPACITY_DESCRIPTION
```

Comment:
These fields are different depending on which BallsDex fork you use. If you use CarFigures, these fields won't work. I would recommend to delete this section as you can use the `LIST` command.

Reply:
Fair enough.